var/spack/repos/builtin/packages/slate/package.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Slate(Package):
+    """The Software for Linear Algebra Targeting Exascale (SLATE) project is
+    to provide fundamental dense linear algebra capabilities to the US
+    Department of Energy and to the high-performance computing (HPC) community
+    at large. To this end, SLATE will provide basic dense matrix operations
+    (e.g., matrix multiplication, rank-k update, triangular solve), linear
+    systems solvers, least square solvers, singular value and eigenvalue
+    solvers."""
+
+    homepage = "https://icl.utk.edu/slate/"
+    hg      = "https://bitbucket.org/icl/slate"
+    maintainers = ['G-Ragghianti']
+
+    version('develop', hg=hg)
+
+    variant('cuda',   default=True, description='Build with CUDA support.')
+    # variant('mkl',  default=True, description='Build using Intel MKL.')
```

Comment:
@G-Ragghianti are you keeping this variant around for the future?

Likewise, will this package possibly make use of cblas/scalapack in the future?

When commenting out variants/deps, it's usually useful to mention what they would be useful for.

Reply:
Right now, MKL is required, but it will eventually be an optional variant.  This is currently where we are getting scalapack from for some of the comparison tests (not required for SLATE functionality).  I will add a comment about this.